src/main/java/com/flipkart/ranger/healthcheck/HealthChecker.java

```diff
@@ -41,7 +45,7 @@ public void run() {
             try {
                 healthcheckStatus = healthcheck.check();
             } catch (Throwable t) {
-                logger.error("Error running healthcheck. Setting node to unhealthy");
+                logger.error("Error running healthcheck. Setting node to unhealthy", t);
                 healthcheckStatus = HealthcheckStatus.unhealthy;
             }
             if(HealthcheckStatus.unhealthy == healthcheckStatus) {
```

Comment:
unhealthy should be enum and hence unhealthy becomes a final and static constant.